bcl_manifest.json

```diff
@@ -44,5 +44,10 @@
     "org": "NREL",
     "type": "measure",
     "url": "https://github.com/NREL/openstudio-load-flexibility-measures-gem"
+  }, {
+    "name": "Automatic Magic Measures",
+    "org": "automaticmagic",
+    "type": "measure",
+    "url": "github.com/automaticmagic/automaticmagic-measures"
```

Comment:
```suggestion
    "url": "https://github.com/automaticmagic/automaticmagic-measures"
```

Reply:
Also, seems like your lib folder is different than mine? https://github.com/jmarrec/openstudio-effibem-measures-gem/tree/main/lib or an NREL one: https://github.com/NREL/openstudio-load-flexibility-measures-gem/tree/master/lib

https://github.com/automaticmagic/automaticmagic-measures/tree/main/lib